### DISCUSSION TITLE
Does Socket.io is working this that version

### DISCUSSION THREAD

**Discussion by shishamo** [fastify/fastify#4143] [Q&A] [answered]
Does someone knows if that version is working with Socket.io?
I've tried in a previous minor version and the Socket.io plugin was not working yet, only with the v3 of fastify.
Thank you

**Comment by climba03003** [comment, marked as answer]
What do you means by `this` version, `that` version and `previous minor` version?
Please specify the version with number.

If you are talking about why the socket cannot be connected in `fastify@4`, please specify the `host` explicitly and do not use `localhost`.
For example:
```js
import Fastify from 'fastify'

const fastify = Fastify()
fastify.listen({
  port: 3000,
  host: '127.0.0.1' // here is important, it disable the double bindings
})
```
